dashboard/config/locales/long_instructions.en.yml

```diff
@@ -6982,7 +6986,7 @@ en:
 
         * In the Animation Tab are animations for each of your sprites. Go look at what they are.
         * In your code give each sprite its appropriate animation. **Use the ones provided for now** but later you'll be able to go change them.
-        * Head to [Level 4](/s/csd3-2018/stage/20/puzzle/4) if you need help remembering how to do this.
+        * Head to [Level 4](../../20/puzzle/4) if you need help remembering how to do this.
```

Comment:
I know this technically _works_, but it always weirds me out to see URLs constructed this way